docs/release-notes/NuGet-7.0.md

```diff
@@ -1,26 +1,303 @@
 ---
 title: NuGet 7.0 Release Notes
 description: Release notes for NuGet 7.0 including new features, bug fixes, and DCRs.
-author: zivkan
+author: donnie-msft
+ms.date: 10/29/2025
 ms.topic: release-notes
 ---
-# NuGet 7.0 Release Notes
 
-<!--
-    This is intentionally not yet added to TOC.md.
-    When this version is ready for release:
-    1. Rename this file to the next version
-    2. Change the version strings in this file to the next version
-    3. Use the release tool to create the real release notes for the version ready for release
-    4. Add the release notes to TOC.yml
--->
+# NuGet 7.0 Release Notes
 
 NuGet distribution vehicles:
 
 | NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
-|:---|:---|:---|
-| [**7.0**](https://nuget.org/downloads) | TBD | TBD |
+| :---|:---|:---|
+| [**7.0.0**](https://nuget.org/downloads) | [Visual Studio 2026 version 18.0.0](https://visualstudio.microsoft.com/downloads/) | [10.0.100](https://dotnet.microsoft.com/download/dotnet/10.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2026 with any .NET workload
+
+## Summary: What's New in 7.0.0
+
+* Created the [NuGet Model Context Protocol (MCP) Server](https://learn.microsoft.com/nuget/concepts/nuget-mcp-server) available in package, [NuGet.Mcp.Server](https://www.nuget.org/packages/NuGet.Mcp.Server#readme-body-tab)
```

Comment:
@jeffkl can you help with this. 
How do we to best advertise thing?
Is it available by default, etc, let's put it in the context of the Visual Studio users if possible.